lib/coprl/presenters/plugins/theme.rb

```diff
@@ -97,7 +24,8 @@ def view_dir_theme(_pom)
           # It will be called once for the page.
           # The pom is passed along with the sinatra render method.
           def render_header_theme(pom, render:)
-            render.call :erb, 'theme_header', views: view_dir_theme(pom)
+            render.call :erb, 'theme_header', views: view_dir_theme(pom),
+                        locals: { theme: pom.context[:current_theme].try(:call) || {} }
```

Comment:
We are now using a lambda to get the current_theme. I can also just pass the theme and it will also work, however as we are resetting the scopes dynamically in some poms, then that needs special logic to also clear and reset the theme every time those methods are invoked. This seemed like a simpler approach